### PR TITLE
Update translations and supported languages

### PR DESCRIPTION
**Pull request by jinhojang6**
Update

1. Status app translations
- German translation by Oleksandr & @minimisthupper
- Chinese translation by Steven
- Russian translation by @flexsurfer 
- Portuguese-Brazilian translation by @3esmit 

2. Play store titles & descriptions
- Chinese(Simplified & Traditional), Indonesian, Turkish, Arabic, Russian, Italian, French, German, Portuguese, Portuguese Brazilian, Spanish, Ukrainian, Tagalog, Vietnamese, and Korean
- English was slightly updated based on https://notes.status.im/r9_8wSs5RbaPGXZoISGjLw

### CODE REVIEW

**Comment by jakubgs** [comment]
I don't see you adding the `tr` translation file:
```
Error: Cannot find module '../translations/tr.json'
```

**Comment by jinhojang6** [comment]
@jakubgs 
I wanted to include this `tr.json` which was preuploaded before
- https://github.com/status-im/status-react/blob/update-translation/translations/tr.json

**Comment by jakubgs** [comment]
OH, you're right, it is there on the CI host:
```
jenkins@linux-04:~/workspace/tatus-react_prs_android_PR-10656$ ls -l translations/ | grep tr
-rw-rw-r-- 1 jenkins jenkins  71933 May 14 16:46 tr.json
```
Wtf...

File: src/status_im/i18n_resources.cljs [hunk 1]
```diff
@@ -25,6 +26,7 @@
    :it         (js/require "../translations/it.json")
    :ko         (js/require "../translations/ko.json")
    :ru         (js/require "../translations/ru.json")
+   :tr         (js/require "../translations/tr.json")
```
Comment (jakubgs) [inline src/status_im/i18n_resources.cljs:30]: This `../` is in relation to what? @yenda maybe you can help us out. Tests are failing here on importing this file.

Comment (yenda) [reply src/status_im/i18n_resources.cljs:30]: Translations also need to be added here for tests https://github.com/status-im/status-react/blob/develop/src/mocks/js_dependencies.cljs#L194

Comment (flexsurfer) [reply src/status_im/i18n_resources.cljs:30]: @jinhojang6

File: src/mocks/js_dependencies.cljs [hunk 2]
```diff
@@ -192,6 +192,7 @@
     "react-native-image-resizer" image-resizer
     "./fleets.js" default-fleets
     "../translations/ar.json" (js/JSON.parse (slurp "./translations/ar.json"))
+    "../translations/en.json" (js/JSON.parse (slurp "./translations/de.json"))
```
Comment (flexsurfer) [inline src/mocks/js_dependencies.cljs]:
```suggestion
    "../translations/de.json" (js/JSON.parse (slurp "./translations/de.json"))
```

File: src/mocks/js_dependencies.cljs [hunk 3]
```diff
@@ -201,6 +202,7 @@
     "../translations/it.json" (js/JSON.parse (slurp "./translations/it.json"))
     "../translations/ko.json" (js/JSON.parse (slurp "./translations/ko.json"))
     "../translations/ru.json" (js/JSON.parse (slurp "./translations/ru.json"))
+    "../translations/ko.json" (js/JSON.parse (slurp "./translations/tr.json"))
```
Comment (flexsurfer) [inline src/mocks/js_dependencies.cljs]:
```suggestion
    "../translations/tr.json" (js/JSON.parse (slurp "./translations/tr.json"))
```

**Review by flexsurfer — APPROVED** [review]

**Comment by jinhojang6** [comment]
It's all good now. Thanks @flexsurfer @yenda  @jakubgs

**Comment by 3esmit** [comment]
I still need to do like 85% of Portuguese Brazilian (PT_BR). I will work on it on weekend so perhaps we can have it done by next week.

**Comment by jinhojang6** [comment]
Okay, I will wait for it @3esmit . Please don’t merge this PR yet

File: src/status_im/i18n_resources.cljs (hunk 1 above)
Comment (jakubgs) [reply src/status_im/i18n_resources.cljs:30]: Should we document this somewhere? @jinhojang6 maybe you could add a comment in the `src/status_im/i18n_resources.cljs` file about the `src/mocks/js_dependencies.cljs` and vice versa?

Comment (jinhojang6) [reply src/status_im/i18n_resources.cljs:30]: Okay, what do you think of adding the comments to both of the files pointing to each other? @jakubgs

Comment (jakubgs) [reply src/status_im/i18n_resources.cljs:30]: Yes, that's exactly what I had in mind.
https://www.merriam-webster.com/dictionary/vice%20versa

Comment (jinhojang6) [reply src/status_im/i18n_resources.cljs:30]: Oh sorry, I read your comment wrong. I read it as ‘or’ which actually was ‘and’.

No problem, will update this PR when Ricardo gives me the Portuguese translations. @jakubgs

**Comment by 3esmit** [comment]
I gone through all 1096 texts in Lokalise and filled the remaining, updated some that were wrong (e.g. in spanish) and updated some that had a better word to describe.

I marked some as "unverified" as I couldn't verify it properly, as the translation depends on context.

Any errors won't be of high impact, and I will identify them while using the DApp in Protuguese-Brazilian myself.

**Comment by jinhojang6** [comment]
I tried to rebuild this PR with new German translation. And the android-e2e failed.

@jakubgs @flexsurfer @yenda

**Comment by jakubgs** [comment]
Unrelated failure due to a Nix bug(https://github.com/NixOS/nix/issues/3593), just re-run it.

**Comment by jinhojang6** [comment]
@jakubgs @flexsurfer @yenda

I've just updated the Play store translations of 16 languages but encountered an error with rebasing. Let me take a look and double-check the translations

**Comment by jakubgs** [comment]
Looks like a rebase is needed:
```
[2020-05-19T17:46:49.366Z] + git rebase develop
[2020-05-19T17:46:49.366Z] First, rewinding head to replay your work on top of it...
[2020-05-19T17:46:49.366Z] Applying: Update translations and supported languages
[2020-05-19T17:46:49.366Z] Using index info to reconstruct a base tree...
[2020-05-19T17:46:49.366Z] M	src/status_im/i18n_resources.cljs
[2020-05-19T17:46:49.366Z] Falling back to patching base and 3-way merge...
[2020-05-19T17:46:49.367Z] Auto-merging src/status_im/i18n_resources.cljs
[2020-05-19T17:46:49.367Z] CONFLICT (content): Merge conflict in src/status_im/i18n_resources.cljs
[2020-05-19T17:46:49.367Z] error: Failed to merge in the changes.
[2020-05-19T17:46:49.367Z] Patch failed at 0001 Update translations and supported languages
```

**Comment by jinhojang6** [comment]
Finally built an apk file. @3esmit can you give this build a try?

File: src/status_im/i18n_resources.cljs (hunk 1 above)
Comment (3esmit) [reply src/status_im/i18n_resources.cljs:30]: There is missing pt_BR in this list.

**Comment by 3esmit** [comment]
It still crashes on the first run after install, but after retrying to open several times, it opened. I found a lowercase in the splash screen "Start" button and I'm going to update it on Lokalise

**Comment by 3esmit** [comment]
I just updated the "get-started" string to use a uppercase letter in the first character. I also checked all other lowercase starting strings to see if they were wrong and all seems good.

I will keep looking for mistakes on the translation, but for now looks good.

**Review by 3esmit — APPROVED** [review]

**Comment by jinhojang6** [comment]
Updated everything.
@flexsurfer Can we test and merge this PR?

**Comment by Serhy** [comment]
Confirmed on iOS three languages loads when device language is set appropriately.

**Pull request merged**